app/controllers/api/v0/results_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Api::V0::ResultsController < Api::V0::ApiController
+  def personal_records
+    user = User.find(params.require(:user_id))
+
+    render json: user.personal_records
```

Comment:
Thanks for the previous set of changes.
Now you've gotta pay attention that this call doesn't return WCIF automatically 🙃

Reply:
Fixed :melt: